.github/workflows/contracts.yml

```diff
@@ -247,7 +243,7 @@ jobs:
       - name: Build and publish initcontainer
         uses: docker/build-push-action@v2
         env:
-          IMAGE_NAME: 'initcontainer-provision-keep-client'
+          IMAGE_NAME: initcontainer-provision-keep-client-${{ github.event.inputs.environment }}
```

Comment:
Renaming the image will require aligning the kubernetes configs.

Reply:
Are those configs stored on the repo? Could you guide me where to find them?

Reply:
Please grep ocurrences under `./infrastructure/kube/keep-dev` and `./infrastructure/kube/keep-test`.

Reply:
I've changed the Kubernetes config. and also modified the image name in the workflow to use chain name in the suffix instead of the environment/network. In order for this to work, https://github.com/keep-network/ci/pull/6 needs to be merged to master, so please merge that PR before merging the current one.